### PR TITLE
LP-2747 - Remove unnecessary queries

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**

### CODE REVIEW

**Review by annas-usman — APPROVED** [review]

File: openedx/adg/lms/applications/models.py [hunk 1]
```diff
@@ -419,7 +419,7 @@ class Meta:
         app_label = 'applications'
 
     def __str__(self):
-        return 'id={id} name={name}'.format(id=self.course.id, name=self.course.display_name)
+        return '{}'.format(self.course_id)
```
Comment (javeriaejaz-arbisoft) [inline openedx/adg/lms/applications/models.py:422]: And f-string here too.

**Review by javeriaejaz-arbisoft — CHANGES_REQUESTED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2298/workflows/workflows_id/jobs/2298/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (50.0%): Missing lines 109,422
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/webinars/models.py (0.0%): Missing lines 330
-------------
Total:   6 lines
Missing: 3 lines
Coverage: 50%
-------------
```

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Comment by MunamMubashir-arbisoft** [comment]
The tests are all running successfully. The failure is because of the coverage where only a couple line change was made in a commit.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2312/workflows/workflows_id/jobs/2312/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Pull request merged**